conda/resolve.py

```diff
@@ -1,5 +1,6 @@
 # Copyright (C) 2012 Anaconda, Inc
 # SPDX-License-Identifier: BSD-3-Clause
+"""The low-level SAT solver wrapper/interface (conda.core.solver.Solver is a higher-level API)."""
```

Comment:
This is only part of the `classic` solver, if that needs to be added.

Reply:
Good idea